templates/chat/item-toHitAoe-card.hbs

```diff
@@ -57,7 +57,7 @@
             <div class="hit-roll">{{{ renderedHitRoll }}}</div>
             <div class="description-medium">{{{ hitRollText }}}</div>
             {{#if (lt hitRollValue 3)}}
-            <img class="hexFacing" src="systems/hero6efoundryvttv2/icons/HexFacing.svg">
+            <img class="hexFacing" src="systems/{{HerosysModule}}/icons/HexFacing.svg">
```

Comment:
Unclear where this is coming from, I haven't loaded this PR.  I presume it works.

Reply:
I think the hexfacing does double duty as the targeting arrows? for when you cover a token with an AoE?

I can make it load something else and see what it does.

Reply:
![missing image](https://github.com/dmdorman/hero6e-foundryvtt/assets/71213987/d63565c9-ee94-4ba6-b76a-b9da17b6e4d8)
Found it, and I broke it. I think it's used to show the direction to move a missed AoE.

Reply:
This is what it should be

![fixed image](https://github.com/dmdorman/hero6e-foundryvtt/assets/71213987/6047f265-c443-4c10-9f72-a532d4c6e150)

Reply:
i can fix it

Reply:
fixed in https://github.com/dmdorman/hero6e-foundryvtt/pull/938/commits/23b1856379b18b1df22bf5e5a6b123b94fa6838c

Reply:
ok, I kinda knew that data already had to be in there somewhere ... it is.

Reloading the system.json is duplicated work; I will remove that part, but the rest should be golden